app/services/appointment_fetcher.rb

```diff
@@ -3,7 +3,8 @@ def call(work_order_reference:)
     @work_order_reference = work_order_reference
 
     appointments = available_appointments
-    appointments.map { |appointment| AppointmentPresenter.new(appointment) }
+    appointments = filter_before_tomorrow(appointments)
+    filter_long_slots(appointments)
```

Comment:
Would this read better as something like:

appointments.map {}
  .select { |a| after_tomorrow(a) }
  .reject { |a| long_slot(a) }

or even

appointments.map {}
  .select(&:after_tomorrow)
  .reject(&:long_slot)  

Maybe not worth it